bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -18,9 +18,13 @@ User.where.not(school_info: nil).where(user_type: "teacher").find_in_batches(bat
         last_confirmation_date: user.created_at
       )
       num_users_updated += 1
+    rescue StandardError => e
+      puts "Error: teacher with user id #{user.id} with school id #{user.school_info.id}"
+      exceptions.push(e)
     end
+    num_users_updated = 0
```

Comment:
I don't think we need to rollback an entire batch because a single attempt to create a SchoolUserInfo failed.  I think it's safe to log the User and SchoolInfo and the error message and move on.

Reply:
Personally, I'd prefer that we make the batch atomic; I understand needing to stop and resume on errors, but it may be more straightforward to restart at a particular user ID, and individual batches should not be long-running queries.

Not to start a tug-of-war on this particular issue, but curious what @uponthesun's thoughts on this are.

Reply:
I don't think I know enough about the two options to have a strong opinion. Are errors somewhat likely / expected, or are they likely to reflect a problem with the script logic?

Are we trying to have the script run to completion as much as possible, and then afterwards manually inspect all the errors, or are we trying to have the script stop running when it encounters an error and fix it before going forward?